tag/context.go

```diff
@@ -45,10 +45,10 @@ type ctxKey struct{}
 var mapCtxKey = ctxKey{}
 
 func init() {
-	exemplar.RegisterAttachmentExtractor(extractTagsAttachments)
+	metricdata.RegisterAttachmentExtractor(extractTagsAttachments)
```

Comment:
This is a wrong usage of exemplars, tags are not "exemplars"

Reply:
Removed.